test/e2e/instrumentation/core_events.go

```diff
@@ -121,7 +123,9 @@ var _ = common.SIGDescribe("Events", func() {
 				break
 			}
 		}
-		framework.ExpectEqual(foundCreatedEvent, false, "should not have found test event after deletion")
+		if foundCreatedEvent {
+			framework.Failf("should not have found test event %s/%s in %+v", eventTestName, f.Namespace.Name, eventsList.Items)
```

Comment:
Same here and below.

Reply:
So should I include name and namespace in the output. If yes should I do like this -
`if !foundCreatedEvent{
         framework.Failf("unable to find test event with test name : %v and namespace name : %v in %+v", eventTestName, f.Namespace.Name, eventsList.Items)
}`

Reply:
` : ` as separator looks ugly. Messages should be plain English, not some made up markup.

```suggestion
			framework.Failf("Should not have found test event %s in namespace %s, full list of events is %+v", eventTestName, f.Namespace.Name, eventsList.Items)
```